frontend/src/components/MessagesPage.vue

```diff
@@ -86,76 +79,78 @@ export default {
         dateReceived: '',
         messageContent: '',
       },
+      displayName: '',
     };
   },
   computed: {
-    ...mapState(useAuthStore, ['userInfo']),
-    ...mapState(useMessageStore, ['allMessages']),
+    userInfo() {
```

Comment:
to keep the code consistent with other pages, can we still use mapState? if we use mapState, we can just use this.allMessages and this.userInfo instead of adding extra computed values